modules/javafx.graphics/src/main/java/com/sun/glass/ui/gtk/screencast/TokenStorage.java

```diff
@@ -365,13 +366,10 @@ static Set<TokenItem> getTokens(List<Rectangle> affectedScreenBounds) {
 
         // 2. Try screens of the same size but in different locations,
         // screens may have been moved while the token is still valid
-        List<Dimension> dimensions =
+        List<Dimension2D> dimensions =
                 affectedScreenBounds
                 .stream()
-                .map(rectangle -> new Dimension(
-                        rectangle.width,
-                        rectangle.height
-                ))
+                .map(rectangle -> new Dimension2D(rectangle.width, rectangle.height))
```

Comment:
the largest positive integer that can be stored in a float exactly is ~16M.  we are fine here.